src/main/business/applicationFacade.js

```diff
@@ -336,6 +350,45 @@ class ApplicationFacade{
         return this.userController.updateUser(data);
     }
 
+
+    addManualMetaQuestionToExam(data) {
+        //todo - implement save the question (it has all the fields)
+        // as of now it just returns the question without saving it to an "current exam" object and so all the
+        // functionality of fetching relevant questions/keys/distractors is not correct
+        return {
+            id: data.question.selectedMetaQuestion.id,
+            stem: data.question.selectedMetaQuestion.stem,
+            appendix: data.question.selectedMetaQuestion.appendix,
+            key: data.question.selectedKey,
+            distractors: data.question.selectedDistractors
+        };
+    }
+
+    addAutomaticQuestionToExam(data) {
```

Comment:
add a call to `validateParameters` so we know what is the received data
maybe @OfekN11 can do it for you

Reply:
This should be (and will be) done at the business level, at the implementation.

Thats due to the possibilty that more function will call it.

The validation should be done by the ones who use the data.